test-files/cmd/slow/ConflateCmdStatsAllDataTypesTest.sh.stdout

```diff
@@ -8,54 +8,53 @@ Percentage of Total Features Conflatable	100	100	0
 Untagged Feature Count	0	0	0	
 Total Unconflatable Features	0	0	0	
 Percentage of Total Features Unconflatable	0	0	0	
-Total Features Processed By Conflation	0	0	21	
 Number of Match Creators	3	3	3	
 Features Conflatable by: hoot::BuildingMatchCreator	10	8	0	
 Features Conflatable by: hoot::PlacesPoiMatchCreator	11	10	0	
 Features Conflatable by: hoot::HighwayMatchCreator	4	4	0	
 Total Conflated Features	0	0	21	
 Percentage of Total Features Conflated	0	0	65.625	
-Total Features Marked for Review	0	0	0	
-Percentage of Total Features Marked for Review	0	0	0	
+Total Features Involved in a Review	0	0	0	
+Percentage of Total Features Involved in a Review	0	0	0	
 Total Number of Reviews to be Made	0	0	0	
 Total Unmatched Features	25	22	11	
 Percentage of Total Features Unmatched	100	100	34.375	
 POI Count	11	10	8	
 Conflatable POIs	11	10	0	
 Conflated POIs	0	0	8	
-POIs Marked for Review	0	0	0	
+POIs Involved in a Review	0	0	0	
 Number of POI Reviews to be Made	0	0	0	
```

Comment:
I think that's one is due to the fact that core still makes PLACES the default poi algorithm, while UI makes Unifying the default....so those poi's don't get marked for review with PLACES.  core probably should switch over to Unifying as its default, and this test actually is more meaningless than it should be if unifying isn't the default....I'll ask Jason what he thinks about finally making the switch.